docker-compose-pool-combo.yaml

```diff
@@ -0,0 +1,206 @@
+      context: .
+      dockerfile: ./enclave_manager/Dockerfile
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+        - ENCLAVE_TYPE=wpe
+        # Configure list of workload that need to be supported
+        # by this WPE by separating each workload by delimiter semicolon.
+        - WORKLOADS=heart-disease-eval
+    environment:
+      - http_proxy
+      - https_proxy
+      - no_proxy
+    volumes:
+      - pool-1:/shared-pool-1
+    expose:
+      # ZMQ socket port.
+      - 5555
+    command: |
+      bash -c "
+        mv /project/avalon/wpe_mr_enclave.txt /shared-pool-1
+        enclave_manager --lmdb_url http://avalon-lmdb:9090 --kme_listener_url http://avalon-kme-1:1948 --worker_id pool-1
+        tail -f /dev/null
+      "
+    depends_on:
+      - avalon-lmdb
+      - avalon-listener
+      - avalon-kme-1
+
+  # Pool-2 KME
+  avalon-kme-2:
+    container_name: avalon-kme-2
+    image: avalon-enclave-manager-kme-dev
+    build:
+      context: .
+      dockerfile: ./enclave_manager/Dockerfile
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+        - ENCLAVE_TYPE=kme
+    environment:
+      - http_proxy
+      - https_proxy
+      - no_proxy
+    volumes:
+      - pool-2:/shared-pool-2
+    expose:
+      # KME listener port
+      - 1948
+    command: |
+      bash -c "
+        while true;
+          do
+            if [ -e /shared-pool-2/wpe_mr_enclave.txt ];
+              then cp /shared-pool-2/wpe_mr_enclave.txt /project/avalon/wpe_mr_enclave.txt;
+              break;
+            fi;
+            sleep 1;
+        done;
+        enclave_manager --lmdb_url http://avalon-lmdb:9090 --bind http://avalon-kme-2:1948 --worker_id pool-2
+        tail -f /dev/null
+      "
+    depends_on:
+      - avalon-lmdb
+      - avalon-listener
+
+  # Pool-2 WPE 1
+  avalon-enclave-manager-2-1:
+    container_name: avalon-wpe-2-1
+    image: avalon-enclave-manager-wpe-echo-dev
+    build:
+      context: .
+      dockerfile: ./enclave_manager/Dockerfile
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+        - ENCLAVE_TYPE=wpe
+        # Configure list of workload that need to be supported
+        # by this WPE by separating each workload by delimiter semicolon.
+        - WORKLOADS=echo-result
+    environment:
+      - http_proxy
+      - https_proxy
+      - no_proxy
+    volumes:
+      - pool-2:/shared-pool-2
+    expose:
+      # ZMQ socket port.
+      - 5555
+    command: |
+      bash -c "
+        mv /project/avalon/wpe_mr_enclave.txt /shared-pool-2
+        enclave_manager --lmdb_url http://avalon-lmdb:9090 --kme_listener_url http://avalon-kme-2:1948 --worker_id pool-2
+        tail -f /dev/null
+      "
+    depends_on:
+      - avalon-lmdb
+      - avalon-listener
+      - avalon-kme-2
+
+  # Pool-2 WPE 2
+  avalon-enclave-manager-2-2:
+    container_name: avalon-wpe-2-2
+    image: avalon-enclave-manager-wpe-echo-dev
+    build:
```

Comment:
We don't need the build section here. We only need to specify the docker image here since this WPE is also part of same worker pool

Reply:
Build section is needed because this service might get executed first in the pool.

Reply:
This comment -> https://github.com/hyperledger/avalon/pull/585/files#r453635740